cmd/gofmtmd/main.go

```diff
@@ -19,10 +19,19 @@ var rootCmd = &cobra.Command{
 	Use:     "gofmtmd",
 	Version: "0.1.1",
 	Short:   "This CLI formats Go Code in Markdown.",
-	Args:    cobra.MinimumNArgs(1),
+	Args:    cobra.MinimumNArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
-		filename := args[0]
-		md, err := ioutil.ReadFile(filename)
+		var filename string
+		if len(args) > 0 {
+			filename = args[0]
+		}
+		var md []byte
+		var err error
+		if filename != "" {
+			md, err = ioutil.ReadFile(filename)
+		} else {
+			md, err = ioutil.ReadAll(os.Stdin)
+		}
```

Comment:
Just decide what to use as io.Reader, how about using ioutil.ReadAll in common? I think this is useful to support other io.Readers in the future

```go
// omit error handling ...

reader := os.Stdin
if filename != "" {
	reader, _ = os.Open(filename) 
} 

md, _ := ioutil.ReadAll(reader)
```